docs/source/qdrant_client.local.tests.rst

```diff
@@ -0,0 +1,18 @@
+qdrant\_client.local.tests package
```

Comment:
not sure if this file should exist

Reply:
This set of files is autogenerated. We don't want to delete it, but this is not rendered in the HTML